tests/Microsoft.Bot.Builder.Dialogs.Tests/DialogExtensionsTests.cs

```diff
@@ -74,6 +74,7 @@ await testFlow.Send("Hi")
                 Assert.IsNotNull(_eocSent, "Skills should send EndConversation to channel");
                 Assert.AreEqual(ActivityTypes.EndOfConversation, _eocSent.Type);
                 Assert.AreEqual("SomeName", _eocSent.Value);
+                Assert.AreEqual("en-us", _eocSent.Locale);
```

Comment:
Just for future refence, it would be good to force and non-default locale and assert for that, en-US almost always works :(

Reply:
OK, modify the create flow function and use en-GB now.